stories/Header.stories.tsx

```diff
@@ -163,4 +163,36 @@ stories
         </Header.Nav>
       </Header>
     );
-  });
+  })
+  .add('Header with custom NavMenuClose and NavItemList component', () => (
+    <Header
+      orgName="Anytown Anyplace"
+      orgSplit="Anywhere"
+      orgDescriptor="NHS Foundation Trust"
+      white
+    >
+      <Header.Container>
+        <Header.Logo href="/" />
+        <Header.Content>
+          <Header.MenuToggle />
+          <Header.Search />
+        </Header.Content>
+      </Header.Container>
+      <Header.NavContainer>
+        <Header.NavTitle>
+          <span>Menu</span>
+          <Header.NavMenuClose />
+        </Header.NavTitle>
+        <Header.NavItemList>
+          <Header.NavItem href="/" mobileOnly>
+            Home
+          </Header.NavItem>
+          <Header.NavItem href="/conditions">Health A-Z</Header.NavItem>
+          <Header.NavItem href="/live-well">Live Well</Header.NavItem>
+          <Header.NavItem href="/social-care-and-support">Care and support</Header.NavItem>
+          <Header.NavItem href="/news">Health news</Header.NavItem>
+          <Header.NavItem href="/service-search">Services near you</Header.NavItem>
+        </Header.NavItemList>
+      </Header.NavContainer>
```

Comment:
This is basically all the changes, you can build your own `Header.Nav` now from the composable sections; and still use the existing `Header.NavItem` like normal :+1: